### DISCUSSION TITLE
How can I hide the name of the navigation tabs on the side bar?

### DISCUSSION THREAD

**Discussion by hellojinwoo** [squidfunk/mkdocs-material#5513] [Q&A]
## How can I hide `Regression` in the side bar?
- Currently my webpage looks as follows: 
![image](https://github.com/squidfunk/mkdocs-material/assets/34431729/220252f3-e4ec-4169-a0e1-0fe1d66c1af0)

- The following is the excerpt of my `mkdocs.yml` file

```
...

theme:
  name: material
  features:
    - navigation.tabs
    - navigation.sections
    - navigation.indexes

...

nav:
  - Home: index.md
  - Regression:
    - Simple Linear Regression:
      - Page 1: linear_regression/01.Parameter_Estimation.ipynb
      - Page 2: linear_regression/02.Overview.ipynb
```

- My question is how I can hide the `Regression` in the side bar, when I clicked the `Regression` in the navigation bar.

**Comment by squidfunk** [comment]
There's no way to do this with configuration. In your case there are no more pages under _Regression_, only a single section, but there might be, which is why this section is kept. In general, the first entry in the navigation bar is always the name of the section, which might be necessary for orientation because the tabs might be hidden. You can use [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css) to hide it, though.
